pom.xml

```diff
@@ -206,6 +208,13 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>com.google.auto.service</groupId>
+                <artifactId>auto-service</artifactId>
+                <version>${auto-service.version}</version>
+                <optional>true</optional>
+            </dependency>
+
```

Comment:
Why introduce this? We don't need this in this PR.

Reply:
I brought it up for testing , It's been deleted.